cmd/authd/daemon/version.go

```diff
@@ -11,8 +11,9 @@ func (a *App) installVersion() {
 	cmd := &cobra.Command{
 		Use:                                                     "version",
 		Short:/*i18n.G(*/ "Returns version of daemon and exits", /*)*/
-		Args:                                                    cobra.NoArgs,
-		RunE:                                                    func(cmd *cobra.Command, args []string) error { return getVersion() },
+
```

Comment:
Nitpick: this blank line is unnecessary

Reply:
It's mostly not to have to split the line

Reply:
Changed this, removing the `i18n.G` and moved as TODO.